pkg/internal/controller/controller.go

```diff
@@ -113,13 +113,14 @@ func (c *Controller) Watch(src source.Source, evthdler handler.EventHandler, prc
 		}
 	}
 
-	c.watches = append(c.watches, watchDescription{src: src, handler: evthdler, predicates: prct})
-	if c.Started {
-		c.Log.Info("Starting EventSource", "source", src)
-		return src.Start(evthdler, c.Queue, prct...)
+	// Controller hasn't started yet, store the watches locally and return.
```

Comment:
Clarify that we're storing the watches so they can all be started when the manager starts this controller

Reply:
Done!